ophyd/flyers.py

```diff
@@ -259,11 +265,26 @@ def _get_stream_name(self, attr):
         obj = getattr(self, attr)
         return self.stream_names.get(attr, obj.name)
 
+    def _describe_with_dtype(self, attr, *, dtype='array'):
```

Comment:
Interesting - I didn't realize that was an option. I'm guessing by your comment you're suggesting it here?